internal/ethapi/api.go

```diff
@@ -656,19 +649,9 @@ func (s *PublicBlockChainAPI) GetProof(ctx context.Context, address common.Addre
 			if storageError != nil {
 				return nil, storageError
 			}
-			// by default, the GetProof API does not return if a storage item
-			// was mutated or not.
```

Comment:
I think this was more readable which should be what we optimize for with new features